lucene/directory-provider/src/test/java/org/infinispan/hibernate/search/TwoNodesTest.java

```diff
@@ -60,6 +60,7 @@ public void testSomething() {
       } finally {
          nodeC.close();
       }
+      waitMembersCount(nodea, SimpleEmail.class, 2);
       assertEquals(2, clusterSize(nodea, SimpleEmail.class));
```

Comment:
We already waited for the cluster size to become 2, no need to assert the same thing again.